core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -119,6 +132,23 @@ public boolean isIgnorePostCommitHooks() {
         return this.ignorePostCommitHooks;
     }
 
+    /**
+     * Data-bound setter for ignoring post commit hooks.
+     *
+     * @param ignorePostCommitHooks
+     *     True if we should ignore post commit hooks, false otherwise.
+     *
+     * @since 2.21
+     */
+    @DataBoundSetter
+    public void setIgnorePostCommitHooks(boolean ignorePostCommitHooks) {
+        this.ignorePostCommitHooks = ignorePostCommitHooks;
+    }
+
+    public String getScmpoll_spec() {
```

Comment:
🐜 not sure what is this method for. The naming is also vague, should be restricted at least